third_party/binutils/test_decenc/drdecode_decenc_x86_test_absrel_s.runcmp

```diff
@@ -0,0 +1 @@
+presence of this file indicates to use runcmp.cmake
```

Comment:
The alternative method of just defining a variable (as I originally suggested over chat) seems simpler and cleaner and avoids all of these files.

Reply:
Done.